gradle-server/src/main/java/com/github/badsyntax/gradle/utils/Utils.java

```diff
@@ -74,4 +76,34 @@ private static String toPackageName(String name) {
 		}
 		return result.toString();
 	}
+	public static Map<String, String> parseArgs(String[] args) {
+		Map<String, String> paramMap = new HashMap<>();
+		for (String arg : args) {
+			if (arg.startsWith("--")) {
+				int index = arg.indexOf('=');
+				if (index != -1) {
+					String key = arg.substring(2, index);
+					String value = arg.substring(index + 1);
+					paramMap.put(key, value);
+				}
+			}
+		}
+		return paramMap;
+	}
+
+	public static int parseIntegerParam(Map<String, String> params, String key) throws IllegalArgumentException {
```

Comment:
It's highly duplicated with `validateRequiredParam`

Just use `Integer.parseInt(validateRequiredParam(xxx))`;